### PR TITLE
docs: Add instructions for resetting password in zitadel

### PR DESCRIPTION
**Pull request by mErrenst**
Closes https://github.com/famedly/product-management/issues/2150

### CODE REVIEW

File: docs/help-center/tim-features/reset-password.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Reset Password</h2>
+    <p class="hero__subtitle">How to reset your password if you have forgotten it or want to change it.</p>
```
Comment (rvanwaveren) [inline docs/help-center/tim-features/reset-password.mdx:17]: Do we need to tell people what are the possible reasons they might want to reset their password? Seems a bit over the top. I think 'how to reset your password' suffices.

File: docs/help-center/tim-features/reset-password.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+</div>
+
+## Change password
+
+To change the password, please follow [these steps](../your-account/account-settings#change-password).
+
+## Reset forgotten password
+
+:::info
+You will only be able to reset your password this way if your account is on a TI-Messenger server. If no new browser window or tab opens for login, you are most likely not on a TI-Messenger server and need to contact your administrator for resetting your password.
```
Comment (rvanwaveren) [inline docs/help-center/tim-features/reset-password.mdx:28]: Can't we just say: If 'reset your password' is not a clickable option under the password input field, you are not on a TIM Server and should contact your administrator to reset your password?

File: docs/help-center/tim-features/reset-password.mdx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+<Tabs>
+  <TabItem value="desktop" label="Desktop">
+  <ol>
+    <li>Open Famedly web app.</li>
+    <li>Enter your login URL into the corresponding input field.</li>
+    <li>Click <b>Next</b>.</li>
+    <li>A new tab for entering the login credentials will open.</li>
+    <li>Enter your username and click **Continue**.</li>
+    <li>Click <b>Reset password</b>.</li>
+    <li>Check your mails for a mail with the title <b>Reset password</b>.</li>
```
Comment (rvanwaveren) [inline docs/help-center/tim-features/reset-password.mdx:40]: Check your email inbox for ... 
We should probably also mention the Spam folder here.

File: docs/help-center/tim-features/reset-password.mdx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    <li>Click the <b>Reset password</b> button in that mail.</li>
+    <li>Choose a password that applies to the rules below the <b>New password</b> input field.</li>
```
Comment (rvanwaveren) [inline docs/help-center/tim-features/reset-password.mdx:42]: That adheres to the rules or that follows the rules.

File: docs/help-center/tim-features/reset-password.mdx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  </TabItem>
+  <TabItem value="mobile" label="Mobile">
+  <ol>
+    <li>Open Famedly app.</li>
+    <li>Enter your login URL into the corresponding input field.</li>
+    <li>Tap <b>Next</b>.</li>
+    <li>A new tab for entering the login credentials will open.</li>
+    <li>Enter your username and tap **Continue**.</li>
+    <li>Tap <b>Reset password</b>.</li>
+    <li>Check your mails for a mail with the title <b>Reset password</b>.</li>
```
Comment (rvanwaveren) [inline docs/help-center/tim-features/reset-password.mdx:56]: same

File: docs/help-center/tim-features/reset-password.mdx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    <li>Tap the <b>Reset password</b> button in that mail.</li>
+    <li>Choose a password that applies to the rules below the <b>New password</b> input field.</li>
```
Comment (rvanwaveren) [inline docs/help-center/tim-features/reset-password.mdx:58]: same

File: i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/reset-password.mdx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">Passwort zurücksetzen</h2>
+    <p class="hero__subtitle">Wie Sie ihr Passwort zurücksetzen können, wenn Sie es vergessen haben oder es ändern möchten.</p>
```
Comment (rvanwaveren) [inline i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/reset-password.mdx:17]: Siehe ENG version

File: i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/reset-password.mdx [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+</div>
+
+## Passwort ändern
+
+Um Ihr Passwort zu ändern, folgen Sie bitte [diesen Anweisungen](../your-account/account-settings#passwort-ändern).
+
+## Vergessenes Passwort zurücksetzen
+
+:::info
+Sie können Ihr Passwort nur auf diese Art zurücksetzen, wenn Ihr Account auf einem TI-Messenger-Server ist. Falls sich kein Browserfenster oder Tab öffnet, ist Ihr Account wahrscheinlich nicht auf einem TI-Messenger-Server und Sie müssen Ihren Administrator kontaktieren, um ein vergessenes Passwort zurückzusetzen.
```
Comment (rvanwaveren) [inline i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/reset-password.mdx:28]: siehe eng v.

File: i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/reset-password.mdx [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+<Tabs>
+  <TabItem value="desktop" label="Desktop">
+  <ol>
+    <li>Öffnen Sie die Famedly Web App.</li>
+    <li>Geben Sie die Anmelde-URL in das Eingabefeld ein.</li>
+    <li>Klicken Sie auf <b>Weiter</b>.</li>
+    <li>Ein neuer Tab zur Eingabebe der Login-Daten wird sich öffnen.</li>
+    <li>Geben Sie Ihren Benutzernamen ein und klicken Sie auf **Weiter**.</li>
+    <li>Klicken Sie auf <b>Passwort zurücksetzen</b>.</li>
+    <li>Öffnen Sie Ihre Mails und suchen Sie nach einer Mail mit dem Titel <b>Passwort zurücksetzen</b>.</li>
```
Comment (rvanwaveren) [inline i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/reset-password.mdx:40]: Siehe eng v.

File: i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/reset-password.mdx [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+  </TabItem>
+  <TabItem value="mobile" label="Mobil">
+  <ol>
+    <li>Öffnen Sie die Famedly App.</li>
+    <li>Geben Sie die Anmelde-URL in das Eingabefeld ein.</li>
+    <li>Tippen Sie auf <b>Weiter</b>.</li>
+    <li>Ein neuer Tab zur Eingabebe der Login-Daten wird sich öffnen.</li>
+    <li>Geben Sie Ihren Benutzernamen ein und tippen Sie auf **Weiter**.</li>
+    <li>Tippen Sie auf <b>Passwort zurücksetzen</b>.</li>
+    <li>Öffnen Sie Ihre Mails und suchen Sie nach einer Mail mit dem Titel <b>Passwort zurücksetzen</b>.</li>
```
Comment (rvanwaveren) [inline i18n/de/docusaurus-plugin-content-docs/current/help-center/tim-features/reset-password.mdx:56]: same

**Review by rvanwaveren — CHANGES_REQUESTED** [review]

File: docs/help-center/tim-features/reset-password.mdx (hunk 4 above)
Comment (mErrenst) [reply docs/help-center/tim-features/reset-password.mdx:42]: I asked DeepL and it suggests "...that complies to the rules". This sounds better to me

File: docs/help-center/tim-features/reset-password.mdx (hunk 2 above)
Comment (mErrenst) [reply docs/help-center/tim-features/reset-password.mdx]: I think it makes more sense to have the opening browser window or tab as indicator. This is a 100% clear. If we use the nonappearance of a button as indicator, users might search for that button in vain and be annoyed.
Can you recheck if my suggestion on this is ok for you?

**Review by rvanwaveren — APPROVED** [review]

**Pull request merged**